src/styles/polygons/polygons_fragment.glsl

```diff
@@ -35,9 +40,14 @@ void main (void) {
     vec4 color = v_color;
     vec3 normal = TANGRAM_NORMAL;
 
-    // Normal material texture (fragment lighting only)
-    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
-        calculateNormal(normal);
```

Comment:
@karimnaaji I followed your suggestion and this should be fixed here (verified in Tangram Play): https://github.com/tangrams/tangram/commit/9113f3aab36a63af7ea9f6b6a18efc1424a17b0d

Maybe someone can double-check this and then we can cut a new release. Thanks for finding!